draft-ietf-privacypass-architecture.md

```diff
@@ -856,23 +869,26 @@ Origin from a token request.
 {{deployment}} discusses deployment models that are possible in practice.
 Beyond possible implications on security and privacy properties of the
 resulting system, Privacy Pass deployments can impact the overall ecosystem
-in two important ways: (1) discriminatory treatment of Clients and the viability
-of an open Web, and (2) centralization. This section describes considerations
-relevant to these topics.
+in two important ways: (1) discriminatory treatment of Clients and the gated
+access to otherwise open services, and (2) centralization. This section
+describes considerations relevant to these topics.
 
 ## Discriminatory Treatment
 
 Origins can use tokens as a signal for distinguishing between Clients
 that are capable of completing attestation with one Attester trusted by the
 Origin's chosen Issuer, and Clients that are not capable of doing the same. A
 consequence of this is that Privacy Pass could enable discriminatory treatment
-of Clients based on Attestation support. This could lead to harmful ecosystem
-effects if left unresolved.
-
-In principle, Issuers should strive to work with a set of Attesters that are
-suitable for all Clients, thereby mitigating such discriminatory behavior.
-In practice, this may require tradeoffs in what type of attestation Issuers are
-willing to trust so as to enable more widespread support.
+of Clients based on Attestation support. For example, an Origin could only
+authorize Clients that successfully authenticate with a token, prohibiting access
+to all other Clients. If the Attestation is not equitable, this can yield
```

Comment:
```suggestion
to all other Clients. If the Attestation is not equally available to all legitimate clients, this can yield
```

Reply:
I don't think we can say this without also defining what legitimate means, and I don't think we have a good definition. That's why I went with equitable.

Reply:
I feel that "equitable" is not necessarily clearly defined here either. That seems to imply some broader statement, as opposed to just saying that "anyone who should be able to pass the bar can". If the use for a particular token is saying "you have a legitimate account to use this privacy proxy or OHTTP relay", then it's OK for that to be limited. If the use is to access a website, it needs to be open to all browsers. Etc, etc.

Reply:
I agree, neither are really clearly defined. What if we just gave examples (like those which you listed)?

Reply:
Sure, maybe giving more concrete examples is the best route here.

Reply:
@tfpauly please take a look at the latest version and let me know what you think?